azkaban-web-server/src/main/java/azkaban/flowtrigger/FlowTriggerService.java

```diff
@@ -344,25 +341,27 @@ public void startTrigger(final FlowTrigger flowTrigger, final String flowId,
       final TriggerInstance triggerInst = createTriggerInstance(flowTrigger, flowId, flowVersion,
           submitUser, project);
 
-      logger.info(String.format("Starting trigger instance[id: %s]", triggerInst.getId()));
+      logger.info(
+          String.format("Starting the flow trigger %s[execId %s] by %s", flowTrigger, triggerInst
```

Comment:
It seems triggerInst.getId() will return the trigger instance id, not the execution id?

Reply:
exec Id is the terminology i used initially to identify an execution of a flow trigger,  but was later changed to trigger instance id. updated the log.